articles/azure-app-configuration/index.yml

```diff
@@ -81,7 +81,7 @@ landingContent:
         - text: Code a Java Spring app
           url: quickstart-java-spring-app.md
         - text: Code a Python app
-          url: quickstart-python.md
+          url: quickstart-python-provider.md
         - text: Azure Resource Manager deployment
           url: quickstart-deployment-overview.md       
```

Comment:
This doesn't belong here.

Reply:
BTW, TOC of this page is also under quickstart, i.e. Quickstart -> Deployment -> Overview.

Reply:
Nothing wrong with the quickstart. It just doesn't belong to "Create and code with Azure App Configuration". Take it out and then you can put in the JS quickstart.

Reply:
sure, I'll do that in a separate PR.